test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -114,58 +120,51 @@ def test_mnist_gpu(sagemaker_session, framework_version, ecr_image, dist_gpu_bac
         pytorch.fit({'training': training_input})
 
 
-@pytest.mark.integration("smmodelparallel")
-@pytest.mark.model("mnist")
-@pytest.mark.processor("gpu")
-@pytest.mark.skip_cpu
-@pytest.mark.skip_py2_containers
-def test_smmodelparallel_mnist_multigpu(ecr_image, instance_type, py_version, sagemaker_session, tmpdir):
```

Comment:
Why is this test removed? Could you add that this has been done on the PR description, and mention why?

Reply:
Removed to simplify. 1 node test for the same script is redundant given that we have a multinode test. I can't think of any added value at least from model parallel library's perspective. If there's any specific reason from DLC testing infra perspective, let me know i can add it back

Reply:
We generally have single-node and multi-node tests to figure out whether the package being tested is broken (single and multi node tests would fail) as opposed to whether there are communication issues in the multi-node case.

Failures in either of the test types are blockers for a merge, and since the feature being tested here is meant to be used in a multi-node setting, I'll leave the decision up to you about whether a single-node test is necessary.